src/domain/event/utils.ts

```diff
@@ -115,3 +118,36 @@ export const downloadEventIcsFile = async ({
     });
   }
 };
+
+export const fetchEvent = async (
+  args: EventQueryVariables,
+  session: ExtendedSession | null
+): Promise<Event> => {
+  try {
+    const { data } = await callGet({
+      session,
+      url: eventPathBuilder(args),
+    });
+    return data;
+  } catch (error) {
+    throw Error(JSON.stringify((error as AxiosError).response?.data));
```

Comment:
Not a massive issue, but looks like this line is uncovered by tests.

Reply:
Added some test to also cover catch block